apps/marketing/src/components/sections/landing/landing.css

```
@@ -0,0 +1,496 @@
+    outline-offset: 3px;
+    border-radius: 4px;
+  }
+
+  /* Primary CTA — bespoke "Start free". Not a flourish: it's the one thing on
+     the page a visitor must reach for, so it's built to feel like the product —
+     a solid, machined key lit from above, grounded by its own colored shadow.
+     Depth is drawn with light (a top highlight, a base seat), not decoration;
+     the only motion is a press that answers the pointer. */
+  & .cta-primary {
+    --cta-lift: 0px;
+    position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)